controllers/controllers.js

```diff
@@ -36,19 +37,34 @@ const getSingleArticleById = (request, response, next) => {
     });
 };
 
-// const getComments = (request, response, next) => {
-//   const { article_id } = request.params;
-//   fetchComments(article_id)
-//   .then((comments) => {
-//     response.status(200).send({ comments: comments });
-//   })
-//   .catch((err) => {
-//     next(err)
-//   })
-// };
+const getComments = (request, response, next) => {
+  const { article_id } = request.params;
+  fetchComments(article_id)
+  .then((comments) => {
+    response.status(200).send({ comments: comments });
+  })
+  .catch((err) => {
+    next(err)
+  })
+};
+
+const postArticle = (req, res, next) => {
```

Comment:
we're posting comments not articles -> please make sure you change your function name